validation-test/stdlib/Prototypes/PersistentVector.swift.gyb

```diff
@@ -403,23 +403,23 @@ struct _PVSparseVectorNodePointer<Key : Hashable, Value>
 
   var childNodePopulationBitmap: _Int32Bitmap {
     unsafeAddress {
-      return UnsafePointer(
-        _nodePointer + _Self._childNodePopulationBitmapOffset)
+      return UnsafePointer((_nodePointer + _Self._childNodePopulationBitmapOffset)
+        .assumingMemoryBound(to: _Int32Bitmap.self))
     }
     nonmutating unsafeMutableAddress {
-      return UnsafeMutablePointer(
-        _nodePointer + _Self._childNodePopulationBitmapOffset)
+      return (_nodePointer + _Self._childNodePopulationBitmapOffset)
+      .assumingMemoryBound(to: _Int32Bitmap.self)
     }
   }
 
   var keyPopulationBitmap: _Int32Bitmap {
     unsafeAddress {
-      return UnsafePointer(
-        _nodePointer + _Self._keyPopulationBitmapOffset)
+      return UnsafePointer((_nodePointer + _Self._keyPopulationBitmapOffset)
+        .assumingMemoryBound(to: _Int32Bitmap.self))
     }
     nonmutating unsafeMutableAddress {
-      return UnsafeMutablePointer(
-        _nodePointer + _Self._keyPopulationBitmapOffset)
+      return (_nodePointer + _Self._keyPopulationBitmapOffset)
+      .assumingMemoryBound(to: _Int32Bitmap.self)
```

Comment:
Please add extra 2 spaces of indentation for the continuation line (here and in a few other places, visible below in this diff).